### DISCUSSION TITLE
Is it possible to search by entry type in the control panel?

### DISCUSSION THREAD

**Discussion by weotch** [craftcms/cms#10710] [Q&A] [answered]
I'd like to search by the Craft entry type.

![Entries - California Olive Ranch CMS 2022-03-08 at 8 39 41 AM](https://user-images.githubusercontent.com/77567/157283678-84771820-cbb3-44f9-a5ad-34df541d9a33.jpg)

I've tried putting `type:bundle` (which is the handle of one of my entry types) into the search bar but I get no results.

![image](https://user-images.githubusercontent.com/77567/157283911-1b09d2f7-ca40-4fa5-8a93-0fcbfe529105.png)

Is this expected behavior?  [The docs](https://craftcms.com/docs/3.x/searching.html#searching-for-specific-element-attributes) don't _mention_ `type` as an "Additional Search Attributes" but, to me, it seems like a pretty natural attribute to make searchable.

**Comment by brandonkelly** [comment, marked as answer]
Not currently, but it’s going to be possible to filter by entry types in Craft 4 (#10075).
